src/utils/getOrganisationDetails.ts

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
+const getOrganisationDetails = (
+  data: Record<string, any>[],
+  isDetails: boolean = false,
+): { organisationValue: string } => {
+  if (Array.isArray(data) && data.length > 0) {
+    const rolesOrder: string[] = ['custodian', 'pointOfContact', 'originator', 'distributor', 'owner'];
+
+    const getOrganisation = (role: string): string => {
+      const obj = data.find((item: Record<string, any>) => item.role === role);
+      return obj ? obj?.organisationObject?.default ?? '' : '';
```

Comment:
ternary operator doesn't add any value here as you are doing safe reading.

Reply:
Fixed